src/MainWindow.cpp

```diff
@@ -82,8 +84,8 @@ void MainWindow::init()
     tabifyDockWidget(ui->dockLog, ui->dockSchema);
     tabifyDockWidget(ui->dockLog, ui->dockRemote);
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0) //Needed only on macOS
```

Comment:
This seems to have dropped the check for OSX?  eg it looks like this will be used for all OS's >= Qt 5.4.0.

Am I reading that wrong?